internal/engine/upper_test.go

```diff
@@ -1679,12 +1679,10 @@ func TestPodUnexpectedContainerAfterSuccessfulUpdate(t *testing.T) {
 		StartTime:    f.Now(),
 		SpanID:       spanID0,
 	})
-	podStartTime := f.Now()
```

Comment:
this isn't a new problem, but is aggravated by the recent timecmp changes. we want to use time.Now() when the pod builder Build(), not define it upfront